doc/linux.t2t

```diff
@@ -210,10 +210,40 @@ Build and install with ninja:
 ninja (uses all cores by default; also supports the above described -jX option)
 ninja install
 ```
-You can build just the targets you need using, for example:
+
+To build even faster, you can build just the targets you need using, for example:
 ```
 ninja qgis
 ninja pycore
+# if it's on desktop related code only:
+ninja qgis_desktop
+```
+
+== Compiling with 3D ==
+
+QGIS 3D requires Qt53DExtras. Unfortunately, these headers have been removed
```

Comment:
Can you split generic and debian specific instructions?

Reply:
To be honest, I'm not sure which ones are for Debian only or other platform. This was just my feedback from Ubuntu.

Reply:
Gentle ping @m-kuhn during this remote code sprint ;-) 
I compile QGIS only on Ubuntu.

Reply:
WITH_3D is generic.
Everything else is Debian specific. Just make a subsection with debian in the title.
And I would remove the term "Unfotunately", I don't think this adds much value for the reader.

Reply:
Done @m-kuhn